doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160.rst

```diff
@@ -438,6 +438,18 @@ To build with the |nRFVSC|, specify ``-DSNIPPET="nrf91-modem-trace-uart" [...]``
 
 See :ref:`cmake_options` for more details.
 
+.. _tfm_enable_share_uart:
+
+Shared UART for application and TFM logging
+====================================================
+
+If you intend to activate tfm logging while having modem traces enabled, it can be useful to direct the tfm logs to the same UART (UART0) as the
+application utilizes. Adding both the ``nrf91-modem-trace-uart`` and ``tfm-enable-share-uart`` snippets can be done in the following way:
```

Comment:
```suggestion
If you want to activate TF-M logging while having modem traces enabled, it can be useful to direct the TF-M logs to the UART (UART0) used by the application.
To activate both modem traces and TF-M logs, use the following command:
```